apps/settings/style/icons.css

```diff
@@ -253,82 +255,3 @@ aside.connecting {
 .bluetooth-type-video-display:before {
   background-position: -36rem -3rem;
 }
-
-
-/******************************************************************************
- * Privacy icons
- */
-
-.privacy-menuitem {
-  -moz-padding-start: 7.4rem;
-  background-repeat: no-repeat;
-  background-position: 1.4rem center;
-}
-
-.privacy-menuitem:before {
-  content: "";
-  width: 3.2rem;
-  height: 3.3rem;
-  background: url(/shared/resources/branding/privacy_sprite.png) no-repeat 3rem 3rem / 9.6rem;
-  position: absolute;
-  left: 1.5rem;
-  top: 50%;
-  margin-top: -1.5rem;
-}
-
-.privacy-browserOS:before {
```

Comment:
It's not related to privacy panel... it's been used in `about_your_privacy.html` so we need keep it